web/sites/default/config/views.view.search_index.yml

```diff
@@ -84,12 +86,13 @@ display:
             'entity:media':
               document: search_result
             'entity:node':
+              contract: default
```

Comment:
Looks like contract was picked up in this view, we should set this to teaser.

Reply:
done